txppb/__init__.py

```diff
@@ -14,7 +14,8 @@ class _FinishLoop(Exception):
 @defer.inlineCallbacks
 def main_loop(
         reactor: interfaces.IReactorTime,
-        engine: ppb.engine.GameEngine
+        engine: ppb.engine.GameEngine,
+        frequency: ppb.engine.GameEngine=None,
```

Comment:
Pretty sure this is supposed to be int or float?

Reply:
ugh, egg on my face. of course. Fixed!